assets/components/manageContributionCta/manageContributionCta.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
```

Comment:
Slightly related to this PR but currently, there are like six different ctas components, Is it worth to think about grouping all of them in a common folder?

Reply:
This one may not need to be a separate component, it's essentially a `CtaLink` with a wrapper `div`. If it's only used in one place, could it be a direct call to `CtaLink` in that place @lmath?

Reply:
@Ap0c Yep, that's a better idea.